Technical-Steering-Committee/proposal.md

```diff
@@ -0,0 +1,89 @@
+
+## Operations and Processes
+
+It is expected that the TSC will establish it's general operations and policies, clearly defined in the [GOVERNANCE.md](../GOVERNANCE.md), but in general the TSC will do the following.
+
+### Chairperson and Secretary
+
+The TSC will elect a chairperson amongst the TSC voting members which will have the responsibilities and duties of...
+
+- Organizing and conducting meetings, and coordinating responses to any voting issues.  
+- Be the primary representative of the Zowe project to the Open Mainframe Project Techincal Advisory Counsel (TAC).
+- Be the primary spokesperson for the Zowe project.
+
+The TSC chairperson may choose from time-to-time to delegate these duties to other TSC voting members if they are unable to fulfill these duties, or if they deem another SME is more appropriate.
+
+The TSC will also appoint a secretary, or chose to have a rotating secretary duty, to record meeting notes, manage actions from meetings and distribute meeting notes to the larger community. 
+
+### Meetings
+
+The TSC will hold a regular cadence of meetings, with the specific day/time/duration to be decided upon by TSC voting members and communicated publicly. TSC meeting agendas will be socialized 1 day prior to the meeting, and any one will be able to add meeting agenda items using the mechanism determined by the TSC voting members. It is encouraged to use tools that enable participation by the community ( such as Slack, GitHub Issues, or mailing lists ). It is generally advised that the TSC maintain a consistent cadence of meetings, and advise the community if meeting times change. TSC meetings by default are open to anyone to participate, but the TSC may hold closed sessions that include Open Mainframe Project staff on an exception basis to discuss sensitive issues such as security vulnerabilities or community conflicts.
+
+All TSC meetings will have recorded notes, and may chose to record the meeting.
+
+### Voting
+
+All general topics and direction will be discussed openly. 
+
+While the Project aims to operate as a [consensus based community](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making), if any TSC decision requires a vote to move the Project forward, the TSC voting members will vote on a one vote per voting member basis. Quorum for TSC meetings is at least fifty percent ( 50% ) of TSC voting members. TSC decisions may be by electronic vote in a method that will be transparent and that can be preserved, such as mailing lists or Github issues. The TSC may choose to have the Open Mainframe Project staff oversee votes to ensure vendor-neutrality. 
+
+In the event a vote cannot be resolved by the TSC, any TSC voting member may refer the matter to the Open Mainframe Project staff and/or Open Mainframe Project TAC for assistance in reaching a resolution.
```

Comment:
I simplified this quite a bit, and ensured it aligned with the voting standards in typical Technical Charters.